src/sentry/middleware/stats.py

```diff
@@ -5,6 +5,7 @@
 import time
 
 from django.http import Http404
+from django.conf import settings
```

Comment:
Nit: Sorting

Reply:
i wouldn't worry about this, we don't have import sorting enforced yet and a bunch of files are relatively unsorted

Reply:
It's still generally nicer if people can sort things. I assume we plan to get isort in at some point?

Reply:
Yeah, soon (tm).